### DISCUSSION TITLE
Geophoto orientation

### DISCUSSION THREAD

**Discussion by nonrev69** [domlysz/BlenderGIS#679] [Q&A]
Hi, has anyone got any experience using the geophoto section I have tried to use it but my cameras a not pointing in the correct orienatation.. as you can see hopefully the image below the camera is pointing due south instead of at the cricle its location is correction though.

I have a feeling this could be due to the drone compass not being calibrated or similar.. the drone I used was a mavic 2 pro.

thanks for any information.

S

![image](https://user-images.githubusercontent.com/72698976/174277337-aecd5b04-6aeb-4513-9a50-ceb86d609bd7.png)

**Comment by CharlieTemplar** [comment]
The geophoto only reads basic exif location info from images and this really only includes long-, lat- and altitude.
DJI drones put the data in separate XMP metadata in the image.
I made a feature request, but ended up making my own mod to the plugin to not only read the orientation of the drone, but also the camera gimbal in relation to the drone and all the camera/lens specific settings, all with a little mavic model as the camera (helps visualize).
Really helped with remote planning of subsequent mapping missions and calculating where to position the drone to recreate aerial shots from the 1930s  

See Issue https://github.com/domlysz/BlenderGIS/issues/378

Share/PM a jpg with the exif data and I will see if it is able to extract the orientation correctly for you.
If it works, I might actually put in the effort to finish what was a covid lockdown distraction 😁

**Comment by CharlieTemplar** [comment]
btw altitude is also a bit sketchy in the metadata.  DJI records relative to take off elevation and also absolute altitude which varies depending on atmospheric presssure and average sea level

**Reply by nonrev69** [reply]
Thanks Charlie for your reply, when I return to the office, I will share a few of the pics.
